conf/maestro-conf/edge-config-dell5000-demo.yaml

```diff
@@ -29,34 +31,38 @@ platform_readers:
 #     This is only a valid option if you are not using a well-known CA
 #    server_name: "devcloud.wigwag.io"
 var_defs:
+   - key: "SNAP_DATA"
+     value: "/var/snap/pelion-edge/current"
+   - key: "SNAP"
+     value: "/snap/pelion-edge/current"
    - key: "TMP_DIR"
      value: "/tmp"
    - key: "WIGWAG_NODE_PATH"
-     value: "/snap/pelion-edge/current/wigwag/devicejs-core-modules/node_modules"
+     value: "{{SNAP}}/wigwag/devicejs-core-modules/node_modules"
    - key: "WIGWAG_DIR"
-     value: "/snap/pelion-edge/current/wigwag"
+     value: "{{SNAP}}/wigwag"
    - key: "NODE_EXEC"
-     value: "/snap/pelion-edge/current/usr/bin/node"
+     value: "{{SNAP}}/bin/node"
    - key: "DEVICEJS_ROOT"
-     value: "/snap/pelion-edge/current/wigwag/devicejs-ng"
+     value: "{{SNAP}}/wigwag/devicejs-ng"
    - key: "DEVJS_CORE_MODULES"
-     value: "/snap/pelion-edge/current/wigwag/devicejs-core-modules"
+     value: "{{SNAP}}/wigwag/devicejs-core-modules"
    - key: "MAESTRO_RUNNER_DIR"
-     value: "/snap/pelion-edge/current/wigwag/devicejs-core-modules/maestroRunner"
+     value: "{{SNAP}}/wigwag/devicejs-core-modules/maestroRunner"
    - key: "SSL_CERTS_PATH"
-     value: "/snap/pelion-edge/current/userdata/edge_gw_config/.ssl"
+     value: "{{SNAP_DATA}}/userdata/edge_gw_config/.ssl"
    - key: "LOCAL_DEVICEDB_PORT"
      value: 9000
    - key: "LOCAL_DATABASE_STORAGE_DIRECTORY"
-     value: "/snap/pelion-edge/current/userdata/etc/devicejs/db"
+     value: "{{SNAP_DATA}}/userdata/etc/devicejs/db"
    - key: "RELAY_VERSIONS_FILE"
-     value: "/snap/pelion-edge/current/wigwag/etc/versions.json"
+     value: "{{SNAP}}/wigwag/etc/versions.json"
    - key: "FACTORY_VERSIONS_FILE"
-     value: "/mnt/.overlay/factory/wigwag/etc/versions.json"
+     value: "/snap/pelion-edge/x1/wigwag/etc/versions.json"
```

Comment:
What is `x1`?

Reply:
I think it refers to the version we want to use as snap is capable of creating multiple versions if we need to rollback. Currently we only have have version 1.